lib/oslc_lisp.pl

```diff
@@ -20,18 +20,28 @@
 
 :- multifile lisp:funct/3.
 
-lisp:funct(send, [ResourceIRI, PostURI], true) :- !,
+call_and_catch(Goal, R) :-
+  catch_with_backtrace((
+      Goal,
+      R = true
+    ),
+    E,
+    ( print_message(error, E),
```

Comment:
what's print_message? What's the different from debug?

Reply:
`debug` is mainly for topic based printouts using format's notation.
`print_message` is for printing exception messages that can be extended by handlers and integrating various outputs like IDEs

Reply:
don't tell anyone, but i just copied from here: https://www.swi-prolog.org/pldoc/doc_for?object=catch_with_backtrace/3

Reply:
stealing like an artist!